src/Jobs/Recruit.Vacancies.Jobs/Communication/CommunicationMessageDispatcher.cs

```diff
@@ -74,7 +74,7 @@ public async Task Run(CommunicationMessageIdentifier commMsgId)
 
         public async Task SendEmail(CommunicationMessage request)
         {
-            _logger.LogInformation($"Trying to send message of type {request.RequestType} to {request.Recipient.Email}");
+            _logger.LogInformation($"Trying to send message of type {request.RequestType}");
```

Comment:
This actually kills the purpose of the log. Instead can we trim the domain from the email so that this log entry makes any sense. You can create an extension method to trim everything after `@` symbol in the email.